public/app/features/alerting/unified/components/folder-actions/FolderActionsButton.tsx

```diff
@@ -70,24 +81,43 @@ export const FolderBulkActionsButton = ({ folderUID }: Props) => {
       )}
       {canDelete && (
         <Menu.Item
-          label={t('alerting.folder-bulk-actions.delete.button.label', 'Delete rules')}
+          label={t('alerting.folder-bulk-actions.delete.button.label', 'Delete all rules')}
           icon="trash-alt"
           onClick={() => setIsDeleteModalOpen(true)}
           disabled={deleteState.isLoading}
         />
       )}
+      {listView2Enabled && (
+        <>
+          <Menu.Divider />
+          <Menu.Item
+            label={t('alerting.folder-bulk-actions.export.button.label', 'Export rules')}
+            icon="download-alt"
+            onClick={() => {}}
```

Comment:
This is not implemented. Should we remove it from the PR?

Reply:
I want to still implement this, but perhaps in a follow-up PR because the current implementation of the folder export needs a `FolderDTO` and we only have a folder `uid`.